### ISSUE TITLE
New effect: black hole

### ISSUE THREAD

**Issue by hasecilu** [Schneegans/Burn-My-Windows#90]
## The Motivation
Black holes are cool.

## The Solution
In the middle of the window (maybe in a random point) create a black circle then the nearest parts of the windows starts to disintegrate (maybe pixel size preserving color) and start orbiting the black hole to finally be sucked. Now that the window has disappeared how to remove the black hole? Hawking radiation. 
Basically the black hole converts the matter (the window) into energy (maybe a ray of light)

[Example animation](https://youtu.be/kOEDG3j1bjs?t=153)

**Comment by Schneegans** [comment]
Hi there! This sounds like a great idea! Do you have an idea how the inverse animation could look like (for opening windows)?

If anyone wants to implement something like this, just drop a note here. As soon as I start working on this, I'll self-assign the issue.

**Comment by hasecilu** [comment]
I have been thinking but I'm still not sure. 
Maybe a pair of massive objects appear in the screen and starts rotating and attracting each other, at some point their bright increases filling the screen, (white or a blueish color) and a small time after the bright dissipates the new window stays.
[Animation example](https://youtu.be/jsGR6QDX7xY?t=1144)

**Comment by jgarza9788** [comment]
https://www.shadertoy.com/view/lXtczl

**Comment by Schneegans** [comment]
Link should have been this, I think: https://www.shadertoy.com/view/lXcfzN?

Looks very cool, although I do not know how it is supposed to interact with the window texture?

**Comment by jgarza9788** [comment]
Maybe similar to portal ?
Where the window comes out of or gets sucked into the black hole

**Comment by Schneegans** [comment]
I have no idea how to do this in a shader, but a more intense, suction-like effect would be nice. Here's a quick Blender mockup: 

https://github.com/user-attachments/assets/db38d272-34e4-43e0-9f01-5dfbffc5700e

Maybe a bit like your shader [here](https://www.shadertoy.com/view/Wcj3Dt) but more sucked into the center. Maybe we could add some wrinkle-like distortions. But it would be cool if the corners are sucked in a bit later than the sides, like in the animation above. This would give a bit of a organic feel, I think.

**Comment by jgarza9788** [comment]
We can't be the first ones to try to do a crumple paper like effect. I bet there is glsl out there on the internet

**Comment by Schneegans** [comment]
That's right. Although a "simple" crumpling is not really what we are looking for, I guess (this would be cool nonetheless, see #93). I am more looking for "tucking a piece of cloth through a hole" 😄 

The animation from above looks like this from the side:

https://github.com/user-attachments/assets/ed0bfa23-5a26-41e3-9e83-5648077ebd5b
